app/modules/assets/models.py

```diff
@@ -282,9 +282,8 @@ def get_symlink(self):
 
     def get_derived_path(self, format):
         git_store_path = pathlib.Path(self.git_store.get_absolute_path())
-        assets_path = git_store_path / '_assets'
         filename = f'{self.guid}.{format}.{self.DERIVED_EXTENSION}'
-        return assets_path / 'derived' / filename
+        return git_store_path / '_derived' / filename
```

Comment:
Is this change going to mean that everyone will need to do a volume wipe when this goes in as the location of the derived file will have changed?

Reply:
No.  The old folder will simply stop being used and the new location will be populated if a derived asset image is needed.  I'm adding an automatic cleanup to remove the old location so it doesn't have any cruft.  This happens anytime we execute `ensure_repository` on a GitStore's repository.